relay-dynamic-config/src/metrics.rs

```diff
@@ -242,6 +344,16 @@ impl MetricExtractionConfig {
     }
 }
 
+/// Configures metrics extraction templates.
+///
+/// Project configs can enable or disable globally defined templates.
+#[derive(Clone, Default, Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct MetricExtractionTemplateOverride {
+    /// `true` if a template should be enabled.
+    pub is_enabled: bool,
+}
```

Comment:
Do we want to support more complicated overriding in the future?

While I generally like the idea it feels like we're building custom override logic for every sub-feature independently (compare generic inbound filters).

Apart from this, even if we only have it for enabled/disabled now, it's a good idea to already have the object in place and it works nicely with default enabled/disabled templates 👍 .

Reply:
> Do we want to support more complicated overriding in the future?

We might!

> While I generally like the idea it feels like we're building custom override logic for every sub-feature independently (compare generic inbound filters).

We might be able to make that code generic (anything that implements an `is_enabled()` trait method can be merged), but generic inbound filters have a bit more merging logic IIRC.